src/client/extensionActivation.ts

```diff
@@ -87,11 +87,12 @@ async function activateLegacy(
     );
     serviceManager.addSingletonInstance<boolean>(IsCodeSpace, env.uiKind == UIKind.Web);
 
-    // Log versions of extensions.
+    // Log env info.
+    standardOutputChannel.appendLine(`${env.appName} (${version}, ${env.remoteName}, ${env.appHost})`);
```

Comment:
We tend to ask this.
Logging most of this in output panel will avoid the need to ask for this information.